moveit_ros/perception/depth_image_octomap_updater/include/moveit/depth_image_octomap_updater/depth_image_octomap_updater.h

```diff
@@ -87,6 +89,8 @@ class DepthImageOctomapUpdater : public OccupancyMapUpdater
   double shadow_threshold_;
   double padding_scale_;
   double padding_offset_;
+  // setting default 0 in order to ignore param
+  double max_update_rate_ = 0;
```

Comment:
I only saw this now, sorry.
Please move the initialization to the cpp files, where everything else is initialized too. (for both updaters)

Reply:
I did this because all of the other parameters were also initialized in the header file.
Why are we shifting this to cpp?

Reply:
:confused: I'm talking about the ` = 0`, right?
All parameters are initialized with default values in the cpp as I see it and you specified the default value in the header for no reason.

The parameter should of course be *defined* in the header.